libcst/_excep.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+class CSTLogicError(Exception):
+    pass
```

Comment:
This should go in `_exceptions.py`, and please write a docstring

Reply:
Thanks for the review !

I'm going to need help on this one though:

- It's in a separate `_excep.py` file, since `_exceptions.py` already imports internals, which in turn may raise a `CSTLogicError`.  
-

Reply:
@zsol 
Any updates ?

Reply:
> _exceptions.py already imports internals, which in turn may raise a CSTLogicError.

Let's move `get_expected_str` out of `_exceptions.py`. That should help.

> For the docstring, I couldn't think of anything meaningful

How about:
```
General purpose internal error within LibCST itself.
```